math/abs_test.go

```diff
@@ -0,0 +1,72 @@
+package math
+
+import (
+	"github.com/TheAlgorithms/Go/math/binary"
+	"math"
+	"testing"
+)
+
+func TestABS(t *testing.T) {
+	tests := getTests()
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			if got := Abs(test.n); got != test.want {
+				t.Errorf("Abs() = %v, want %v", got, test.want)
+			}
+		})
+	}
+}
+
+func TestBinaryABS(t *testing.T) {
+	tests := getTests()
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			if got := binary.Abs(64, test.n); got != test.want {
+				t.Errorf("Abs() = %v, want %v", got, test.want)
+			}
+		})
+	}
+}
```

Comment:
Why did you move this here when `abs.Binary` is not in the package? This does not belong in this package. Move it inside the binary package.

Reply:
Done. is everything ok now?